### PR TITLE
Homework

### PR DESCRIPTION
**Pull request by 123Boryana**

### CODE REVIEW

File: src/main/java/training/day2/HomeworkExercise1.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package training.day2;
+
+public class HomeworkExercise1 {
+    public static void main(String[] args){
+        String[] words = {"alpha", "avatar", "trouble", "birch", "salon"};
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise1.java:6]: Can add space before the variable.

File: src/main/java/training/day2/HomeworkExercise1.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package training.day2;
+
+public class HomeworkExercise1 {
+    public static void main(String[] args){
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise1.java]: Space between ) and {.

File: src/main/java/training/day2/HomeworkExercise1.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        String[] words = {"alpha", "avatar", "trouble", "birch", "salon"};
+
+        for(String str : words){
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise1.java]: Space between ) and {. And even after for. This is for better formatting of the code.

File: src/main/java/training/day2/HomeworkExercise1.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+            if(str.startsWith("a"))
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise1.java]: And even after if. This is for better formatting of the code.

File: src/main/java/training/day2/HomeworkExercise2.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package training.day2;
+
+import java.util.Scanner;
+
+public class HomeworkExercise2 {
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise2.java:8]: Can add space before the variable.

File: src/main/java/training/day2/HomeworkExercise2.java [hunk 6]
```diff
@@ -0,0 +1,20 @@
+package training.day2;
+
+import java.util.Scanner;
+
+public class HomeworkExercise2 {
+    public static void main(String[] args){
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise2.java]: Space between ) and {.

File: src/main/java/training/day2/HomeworkExercise2.java [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter the size of the array: ");
+        int[] array = new int[scanner.nextInt()];
+        int sum = 0;
+        for(int i = 0; i < array.length; i++){
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise2.java]: Add spaces based on the previous remarks.

File: src/main/java/training/day2/HomeworkExercise2.java [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+            System.out.println("Enter your number: " );
+            array[i] = scanner.nextInt();
+            sum += array[i];
+        }
+
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise2.java]: Unnecessary empty line.

File: src/main/java/training/day2/HomeworkExercise3.java [hunk 9]
```diff
@@ -0,0 +1,26 @@
+package training.day2;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+public class HomeworkExercise3 {
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+        List<String> words = new ArrayList<>();
+
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise3.java:11]: Unnecessary empty line.

File: src/main/java/training/day2/HomeworkExercise3.java [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+
+        for(int i = 0; i < 4; i++ ){
+            System.out.println("Type in your word: ");
+            words.add(scanner.next());
+        }
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise3.java:15]: Add empty line.

File: src/main/java/training/day2/HomeworkExercise3.java [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+        List<String> filteredwWords = new ArrayList<>();
+        for(String str : words){
+            if(str.length() >= 4)
+                filteredwWords.add(str);
+        }
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise3.java:21]: Add empty line.

File: src/main/java/training/day2/HomeworkExercise3.java [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+        if(filteredwWords.isEmpty())
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkExercise3.java]: Add space after if. Good formatting. :)

File: src/main/java/training/day2/HomeworkFromMonday.java [hunk 13]
```diff
@@ -0,0 +1,37 @@
+package training.day2;
+
+import java.util.Scanner;
+public class HomeworkFromMonday {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int day = scanner.nextInt();
+
+        if(day >= 1 && day <= 7){
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkFromMonday.java]: When using case, such checking is not needed. Add "default" instead. The switch will check every possible case. If none of the cases is affirmative, then the default will execute. So, remove the if statement and add default.

File: src/main/java/training/day2/HomeworkFromMonday.java [hunk 14]
```diff
@@ -0,0 +1,37 @@
+package training.day2;
+
+import java.util.Scanner;
+public class HomeworkFromMonday {
+    public static void main(String[] args) {
```
Comment (Evgeni-Georgiev) [inline src/main/java/training/day2/HomeworkFromMonday.java:5]: Add new line after the main method.

**Review by Evgeni-Georgiev — CHANGES_REQUESTED** [review]
Excellent.

**Review by Evgeni-Georgiev — APPROVED** [review]
Good.

**Review by intro-java — APPROVED** [review]

**Pull request merged**